test/run_tests.py

```diff
@@ -669,6 +669,8 @@ def filter_csv( values, csv ):
 
     [ 'scale_row_col', gen + dtype + mn + equed + nonuniform_nb + ge_matrix ],
 
+    [ 'matgen', gen_no_nb + dtype + ' --dim 20 --nb 4 --verbose 4' + ge_matrix ],
+
```

Comment:
I updated this so it tested only 20x20 matrix, nb 4. We need to revise `run_tests.py`, or use a different script, to save output to `out` and compare to `ref`.